apps/web/src/pages/CommunityInterests/sections/TrainingAndDevelopmentOpportunities.tsx

```diff
@@ -0,0 +1,285 @@
+import { useIntl } from "react-intl";
+import RectangleGroupIcon from "@heroicons/react/24/outline/RectangleGroupIcon";
+import { useFormContext } from "react-hook-form";
+
+import { CardSeparator, Chip, Chips, Heading } from "@gc-digital-talent/ui";
+import { commonMessages, errorMessages } from "@gc-digital-talent/i18n";
+import {
+  DevelopmentProgramParticipationStatus,
+  FragmentType,
+  getFragment,
+  graphql,
+} from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { DATE_SEGMENT, DateInput, RadioGroup } from "@gc-digital-talent/forms";
+
+import { getClassificationName } from "~/utils/poolUtils";
+
+import { FormValues } from "../form";
+
+export const TrainingAndDevelopmentOpportunitiesOptions_Fragment = graphql(
```

Comment:
exported declaration `TrainingAndDevelopmentOpportunitiesOptions_Fragment` not used within other modules
```suggestion
const TrainingAndDevelopmentOpportunitiesOptions_Fragment = graphql(
```